### PR TITLE
Supported Django Versions

### PR DESCRIPTION
**Pull request by mlavin**
First steps towards 1.0 release. Django migrations support. Testing on 1.8rc1. Few updates to remove warnings/deprecations in Django.

### CODE REVIEW

**Comment by mlavin** [comment]
@vkurup would you have a second to look over these changes?

File: .travis.yml [hunk 1]
```diff
@@ -4,10 +4,8 @@ python:
   - "2.7"
```
Comment (vkurup) [inline .travis.yml]: adding `sudo: false' tells Travis to use their container-based system, which I've found is quite a bit faster. Though now that we're removing support for all these environments, we won't even notice :)

Comment (mlavin) [inline .travis.yml]: Good point. Will update to follow this example for maximum caching/speed https://github.com/dstufft/travis-example/blob/master/.travis.yml

File: tox.ini [hunk 2]
```diff
@@ -1,20 +1,13 @@
 [tox]
-envlist = {py27}-dj{14,15,16,17}-rsms{13,14,15,16,17,18,19}
+envlist = {py27}-dj{17,18}-rsms{18,19}
```
Comment (vkurup) [inline tox.ini]: Might be nice to add environments for flake8 and coverage, but certainly not needed in this PR.

**Comment by vkurup** [comment]
:shipit:

**Comment by mlavin** [comment]
Going to merge and make another PR with tox/Travis improvements.

**Pull request merged**